resilience4j-micronaut/build.gradle

```diff
@@ -0,0 +1,57 @@
+apply plugin: 'groovy'
+repositories {
+    maven { url "https://repo.grails.org/grails/core" }
+    maven { url "https://oss.jfrog.org/oss-snapshot-local" }
+}
+dependencies {
+    annotationProcessor(platform(libraries.micronaut))
+    testAnnotationProcessor(platform(libraries.micronaut))
+    implementation(platform(libraries.micronaut))
+    annotationProcessor "io.micronaut:micronaut-inject-java"
+
+    compileOnly "io.micronaut:micronaut-http"
+    compileOnly "io.micronaut:micronaut-http-server"
+    compileOnly "io.micronaut:micronaut-aop"
+
+    implementation "io.micronaut:micronaut-validation"
+    implementation "io.micronaut:micronaut-router"
+
+    compileOnly project(':resilience4j-circuitbreaker')
+    compileOnly project(':resilience4j-ratelimiter')
+    compileOnly project(':resilience4j-timelimiter')
+    compileOnly project(':resilience4j-retry')
+    compileOnly project(':resilience4j-rxjava2')
+    compileOnly project(':resilience4j-bulkhead')
+    compileOnly project(':resilience4j-consumer')
+    compile project(':resilience4j-framework-common')
```

Comment:
Should be implementation (or possibly 'api' with the `java-library` plugin applied)